distributed/dashboard/core.py

```diff
@@ -34,7 +34,9 @@ def listen(self, addr):
                     check_unused_sessions_milliseconds=500,
                     allow_websocket_origin=["*"],
                     use_index=False,
-                    extra_patterns=[(r"/", web.RedirectHandler, {"url": "/status"})],
+                    extra_patterns=[(r"/", web.RedirectHandler, {"url":
+                                                                 self.prefix +
```

Comment:
Will there ever be situations where `self.prefix` has a trailing slash? In that situation, we would end up with `profix//status`.

Perhaps using a joiner like `urllib.parse.urljoin` or `posixpath.join` may be more robust here?